packages/compass-workspaces/src/components/index.tsx

```diff
@@ -110,6 +110,7 @@ const WorkspacesWithSidebar: React.FunctionComponent<
       >
         <div className={sidebarStyles}>{renderSidebar?.()}</div>
         <div className={workspacesStyles}>
+          {/* TODO: Workspace providers here? */}
```

Comment:
Tabs are siblings, you can't render all of them in one place as stacking context behavior will be just overriding them

Reply:
Ah this was a leftover comment. Sorry!